tests/upgrade/integration/generated/post/query - diddoc/diddoc_indy_32.json

```diff
@@ -8,48 +8,48 @@
         "controller": [
             "did:cheqd:testnet:Nqoxg85PKyr9nPstjTpg5b"
         ],
-        "verification_method": [
+        "verificationMethod": [
```

Comment:
Do we get JSON properties back in response? I thought we were only getting Protobuf property names.